docs/src/all_workflows_parameters.md

```diff
@@ -0,0 +1,76 @@
+- `POSITIVITY_CONSTRAINT (default: 0.01)`: When the list of estimands is generated or validated. Treatment variables' rarest configuration should have at least that frequency. For example if the treatment variables are two variants with minor allele A and T respectively. The rarest configuration will be (AA, TT) and should have a frequency of at least `POSITIVITY_CONSTRAINT`.
+- `NB_PCS (default: 6)`: The number of PCA components to extract.
+
+### UK Biobank Specific
+
+For UK Biobank analyses (`COHORT=UKB`).
+
+- `UKB_CONFIG (default: ${projectDir}/assets/ukbconfig.yaml)`: YAML configuration file describing which traits should be extracted and how the population should be subsetted.
+- `UKB_ENCODING_FILE`: If the `TRAITS_DATASET` is encrypted, an encoding file must be provided.
+- `UKB_WITHDRAWAL_LIST`: List of participants withdrawn from the study.
+- `QC_FILE`: Genotyping quality control file from the UK-Biobank study.
+
+## Secondary Options
+
+These are of less importance.
+
+- `SVP (default: false)`: Whether Sieve Variance Plateau correction should be performed.
+- `FLASHPCA_EXCLUSION_REGIONS (default: assets/exclusion_regions_hg19.txt)`: A path to the flashpca special exclusion regions.
+- `MAF_THRESHOLD (default: 0.01)`: Only variants with that minor allele frequency are considered for PCA.
+- `LD_BLOCKS`: A path to pre-identified linkage disequilibrium blocks to be removed from the BED files for PCA. It is good practice to specify `LD_BLOCKS`, as it will remove SNPs correlated with your variants-of-interest before running PCA.
+- `VERBOSITY (default: 0)`: Verbosity level of the the Workflow's processes.
+- `TL_SAVE_EVERY (default: BATCH_SIZE)`: During the estimation process, results are appended to the file in chunks to free memory.
+- `KEEP_IC (default: SVP)`: To save the Influence Curves for each estimate. Depending on the size of your dataset, this can result in very large disk usage.
+
+### Sieve Variance Plateau Options
+
+These are only relevant if you which to apply the experimental sieve variane plateau correction (`SVP=true`).
+
+- **`GRM_NSPLITS`, default: 100**: To fasten GRM computation, it is typically split in batches.
+- **`NB_SVP_ESTIMATORS`, default: 100**: Number of sieve variance estimates per curve. Setting this value to 0 results in skipping sieve variance correction.
+- **`MAX_SVP_THRESHOLD`, default: 0.9**: Variance estimates are computed for tau ranging from 0 to MAX_SVP_THRESHOLD
+- **`PVAL_THRESHOLD`, default: 0.05**: Only results with a p-value below this threshold are considered for Sieve Plateau Variance correction.
+- **`ESTIMATOR_KEY`, default: 1**: Identifies an estimator from `ESTIMATORS_CONFIG`. The p-value for `PVAL_THRESHOLD` is computed using the result from this estimator.
+
+## Simulations Specific
+
+These apply to both null and realistic simulations (`-entry NULL_SIMULATION` or `-entry REALISTIC_SIMULATION`).
```

Comment:
are there example configs for these subworkflows?

Reply:
okay no worries. have you run these workflows through the targene-pipeline or did you do it in a separate analysis?

Reply:
Sorry I'm not sure I follow? These are two named workflows in this project. While there is no example, there is documentation on them: https://targene.github.io/targene-pipeline/previews/PR199/simulations/overview/